src/test/test_main.cpp

```diff
@@ -12,6 +13,8 @@ void setup() {
 
     RUN_TEST(test_msp_receive);
     RUN_TEST(test_msp_send);
+
+    setup_switches();
```

Comment:
Oh you even moved the test execution line out to the external file... prob a good idea. I'll do this for mine next time they need updating.

Reply:
The downside of what I did is that the fail messages get the wrong file name for some reason. The line numbers are still right, you just have to know which file to look in. Not sure if that's fixable or worth bringing the RUN() lines back into test_main.cpp.

Reply:
I think its fine for now.